frontend/apps/core-app/src/navigation/root.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
```

Comment:
do we still need the drawer navigator if there's only one thing to click on?

Reply:
I think we will need some sort of root navigator in the future so it's useful to keep here. Unsure wether it will be a drawer or not (on mobile it probably will be)